# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424826480 28800
#      Tue Feb 24 17:08:00 2015 -0800
# Node ID 6645488466c764e581995b5bc9599767148fca46
# Parent  e9a0bd0b125c742096a4b620f1812f65f012cacf
__opengl_print__.m: Fix typo introduced in 9fc020886ae9.

* __opengl_print__.m: Concatenate '|' character with pipeline{n} inside square
matrix brackets.

diff --git a/scripts/plot/util/private/__opengl_print__.m b/scripts/plot/util/private/__opengl_print__.m
--- a/scripts/plot/util/private/__opengl_print__.m
+++ b/scripts/plot/util/private/__opengl_print__.m
@@ -163,17 +163,17 @@ function opts = __opengl_print__ (opts)
       fprintf ("opengl-pipeline: '%s'\n", pipeline{n});
     endif
 
     if (strcmp (get (opts.figure, "visible"), "on"))
       ## Use toolkits "print_figure" method
       drawnow (gl2ps_device{n}, ['|' pipeline{n}]);
     else
       ## Use OpenGL offscreen rendering with OSMesa
-      __osmesa_print__ (opts.figure, ['|', pipeline{n}), gl2ps_device{n}]);
+      __osmesa_print__ (opts.figure, ['|' pipeline{n}], gl2ps_device{n});
     endif
   endfor
 
   if (! isempty (strfind (opts.devopt, "standalone")))
     opts.latex_standalone (opts);
   endif
 
 endfunction
